style.css

```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,100;1,800&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;600;800&family=Poppins:ital,wght@0,400;0,700;1,100;1,800&display=swap");
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  max-width: 375px;
+}
+
+#header {
+  display: flex;
+  margin-bottom: 20px;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#headline {
+  margin: 0 16px;
+  display: flex;
+  flex-direction: column;
+}
+
+#humburger_menu {
+  margin-right: 12px;
+}
+
+#left_logo {
+  background-image: url("../images/image_geometry.png");
+  background-repeat: no-repeat;
+  background-position: top left;
+  width: 111px;
+  height: 74px;
+}
+
+#me_names {
+  margin-bottom: 20px;
+  font-family: "Playfair Display", serif;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+}
+
+#me_descriptions {
+  margin-bottom: 40px;
+  line-height: 21px;
+  font-size: 14px;
+  letter-spacing: -0.01em;
+}
+
+#start_collab {
+  color: white;
+  text-align: center;
+  padding: 20px;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 24px;
+  background-color: #1a2236;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! You did not fully meet the requirements mentioned by the [previous code reviewer](https://github.com/Derick1530/portfolio/pull/1#discussion_r1067692244):  

-  The `background picture` of the project is not fully deployed, only the top `left` part is visible, and the `bottom` part is still empty.

- Give the correct  `CSS property` as [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=p9tcEI0GfvTLVZ3f-0) to the text part on your `"Start Collaboration"` button, because the `text` is visible `underlined` and `bold`.

- Give `Header` and `Paragraph` texts correct  **size** and **fonts** like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=p9tcEI0GfvTLVZ3f-0), you can see the difference in the picture below =>

#
![image](https://user-images.githubusercontent.com/63915024/212159366-1905b292-7f09-41d7-b4e2-40b9c604ab84.png)
# 
![image](https://user-images.githubusercontent.com/63915024/212159390-c2787a09-8b0c-43ab-94ea-b61d4436879d.png)
#

Reply:
### Please implement my suggestions 🙏🏼🙏🏼🙏🏼